apps/ideaspace/src/components/modules/WorkshoppingPage/CommentsForm/CommentForm.js

```diff
@@ -20,42 +20,51 @@ function CommentForm(props) {
     const text = e.target.value;
     setTextChange(text);
 
-    if (text.trim() == '') {
+    if (textChange.trim().length === 0) {
       setDisabled(true);
     } else {
       setDisabled(false);
     }
   };
 
   const handleSubmit = async (e) => {
-
     e.preventDefault();
-    var data = { author: userData.username, idea_card: selectedCard, text: textChange.trim() };
+    var data = {
+      text: textChange.trim(),
+      idea_card: selectedCard.id.toString(),
+      author: userData.name,
+      user: userData.id.toString(),
+    };
 
     try {
```

Comment:
can we remove the outer try/catch block here? it looks like it just contains another try/catch block, so it doesn't seem to be serving any purpose here